docs/writing-tests/addon-vitest.mdx

```diff
@@ -0,0 +1,346 @@
+1. Inspect your project's Vite and Vitest setup
+    1. If Vite is not installed and you're using the [`nextjs` framework](../get-started/frameworks/nextjs.mdx), it will install and configure Vite for you, as well as `vite-plugin-storybook-nextjs` (necessary to have your Next.js components function in Vitest).
+    1. Otherwise, if Vite is not installed, it will stop and point you to these instructions to continue setting it up in your project.
+    1. If Vite is installed, it will then check for Vitest.
+        1. If Vitest is not installed, it will:
+            1. Install `vitest`, `@vitest/browser`, and `playwright`
+            1. Run `npx playwright install chromium` to install the Chromium browser engine
+            1. Create a Vitest config file (`vitest.config.ts`) and a Vitest setup file (`storybook.setup.ts`)
+        1. If Vitest is installed, it will stop and point you to these instructions to continue setting it up in your project.
+
+<If renderer="vue">
+	<Callout variant="info">
+		If your stories use template-based Vue components, you may need to [alias the `vue` module](https://vuejs.org/guide/scaling-up/tooling#note-on-in-browser-template-compilation) to resolve correctly in the Playwright CT environment. You can do this via the [`ctViteConfig` property](https://playwright.dev/docs/test-components#i-have-a-project-that-already-uses-vite-can-i-reuse-the-config):
+
+		<details>
+			<summary>Example Playwright configuration</summary>
+
+```ts
+// playwright-config.ts
+import { defineConfig } from '@playwright/experimental-ct-vue';
+
+export default defineConfig({
+  ctViteConfig: {
+    resolve: {
+      alias: {
+        vue: 'vue/dist/vue.esm-bundler.js',
+      },
+    },
+  },
+});
+```
+		</details>
+	</Callout>
+</If>
+
+The configuration produced by the `add` command will attempt to set some sensible defaults for your project. However, you may need to adjust the configuration to fit your project's needs. The full configuration options can be found in the [API section](#options), below.
+
+### Example configuration files
+
+Here are configuration files generated by the `add` command. You can use these as a reference when setting up your project.
+
+<details>
+  <summary>Example Vitest setup file</summary>
+
+```ts title="storybook.setup.ts"
+TK
+```
+</details>
+
+<details>
+  <summary>Example Vitest config file</summary>
+
+```ts title="vitest.config.ts"
+import { defineConfig, mergeConfig } from 'vitest/config'
+import viteConfig from '../vite.config'
+import { storybookTest } from '@storybook/experimental-addon-vitest/plugin'
+
+export default mergeConfig(
+  viteConfig,
+  defineConfig({
+    // ... TK
+  })
+)
+```
+</details>
+
+<details>
+  <summary>Example Vitest workspace file</summary>
+
+{/* TODO: Nextjs & SvelteKit examples */}
+```ts title="vitest.workspace.ts"
+import { defineWorkspace } from 'vitest/config'
+import { storybookTest } from '@storybook/experimental-addon-vitest/plugin'
```

Comment:
@kasperpeulen please share how things will look like from the sb add, it should match to make it more relatable to users that might end up coming from the CLI to the docs